### PR TITLE
Updating existing crypto APIs to v1beta2 and adding in Latest Orderbooks

### PR DESCRIPTION
**Pull request by andrewwoood**
This PR updates the existing APIs to `v1beta2` according to the current version of the [historical data docs](https://alpaca.markets/docs/api-references/market-data-api/crypto-pricing-data/historical/).

There are some `v1beta1` APIs that I didn't change. They aren't in the docs anymore, but I left in single XBBOs, multi XBBOs, and news API. Should these be removed?

In moving to `v1beta2`, the new endpoints are capable of both single and multi. Therefore, I updated the multi-version of our APIs to `v1beta2` and didn't touch the single version. Should the `v1beta1`, single versions of each API be removed? For example, `get_latest_crypto_bar` is `v1beta1` while `get_latest_crypto_bars` is `v1beta2`.

### CODE REVIEW

File: alpaca_trade_api/rest.py [hunk 1]
```diff
@@ -948,15 +943,20 @@ def get_crypto_snapshot(self, symbol: str, exchange: str) -> SnapshotV2:
...
+    def get_latest_crypto_snapshots(self, symbols: List[str]) -> SnapshotsV2:
         resp = self.data_get(
             '/crypto/snapshots',
-            data={'exchange': exchange, 'symbols': _join_with_commas(symbols)},
-            api_version='v1beta1')
+            data={'symbols': _join_with_commas(symbols)},
+            api_version='v1beta2')
         return self.response_wrapper(resp['snapshots'], SnapshotsV2)
 
+    def get_latest_crypto_orderbooks(self, symbols: List[str]) -> OrderbookV2:
```
Comment (haxdds) [inline alpaca_trade_api/rest.py:953]: We need a single symbol method as well. `get_latest_crypto_orderbook`.

**Review by haxdds — APPROVED** [review]

File: alpaca_trade_api/rest.py [hunk 2]
```diff
@@ -948,15 +943,20 @@ def get_crypto_snapshot(self, symbol: str, exchange: str) -> SnapshotV2:
...
+        resp = self.data_get(
```
Comment (haxdds) [inline alpaca_trade_api/rest.py:954]: need single symbol method.

**Review by haxdds — CHANGES_REQUESTED** [review]

File: alpaca_trade_api/rest.py [hunk 3]
```diff
@@ -891,13 +878,11 @@ def get_latest_crypto_trade(self, symbol: str, exchange: str) -> TradeV2:
...
         resp = self.data_get(
-            '/crypto/trades/latest',
-            data={'exchange': exchange, 'symbols': _join_with_commas(symbols)},
-            api_version='v1beta1')
+            '/crypto/latest/trades',
+            data={'symbols': _join_with_commas(symbols)},
+            api_version='v1beta2')
         return self.response_wrapper(resp['trades'], LatestTradesV2)
 
     def get_latest_crypto_quote(self, symbol: str, exchange: str) -> QuoteV2:
```
Comment (haxdds) [inline alpaca_trade_api/rest.py:888]: Let's remove the `exchange` parameter from all crypto methods. `exchange` parameter is no longer available on `v1beta2` endpoints.

File: alpaca_trade_api/rest.py [hunk 4]
```diff
@@ -862,11 +853,9 @@ def get_crypto_bars(self,
...
                         end: Optional[str] = None,
-                        limit: int = None,
-                        exchanges: Optional[List[str]] = None,
-                        ) -> BarsV2:
+                        limit: int = None) -> BarsV2:
         return BarsV2(list(self.get_crypto_bars_iter(
-            symbol, timeframe, start, end, limit, exchanges, raw=True)))
+            symbol, timeframe, start, end, limit, raw=True)))
 
     def get_latest_crypto_bar(self, symbol: str, exchange: str) -> BarV2:
```
Comment (haxdds) [inline alpaca_trade_api/rest.py:860]: Same here with the `exchange` parameter.

**Review by haxdds — CHANGES_REQUESTED** [review]
just 1 thing

File: alpaca_trade_api/rest.py (hunk 3 above)
Comment (andrewwoood) [reply alpaca_trade_api/rest.py]: Okay so the functions that still had exchanges were v1beta1 so I removed them. They aren't needed anyway, unless you want separate single-symbol and multi-symbol for each data type (bars, multibars, latest multibars, latest bars).

**Review by haxdds — APPROVED** [review]

**Pull request merged**